desafio-02/lucascorreaads/javascript/desafio02.js

```diff
@@ -0,0 +1,26 @@
+//Programa que lista todos números primos entre 1 e 10000
+
+//Array com os primeiros números primos da lista
+let primos = [2, 3, 5, 7, 11]
```

Comment:
Você pode iniciar primos como [] e remover o for abaixo e seu programa funciona. Verifique e se eu estiver correto, altere o programa.